es6/experimentSetup.js

```diff
@@ -39,4 +39,4 @@ const getExperimentInputs = (experimentName) => {
   return inputArgs;
 };
 
-export default { registerExperimentInput, getExperimentInputs };
+export { registerExperimentInput, getExperimentInputs };
```

Comment:
👍 I'm cool with this. We'll just need to take a note that any exports from this module will now appear in the exposed planout API.